spotfinder/spotfinder.cc

```diff
@@ -903,7 +914,7 @@ int main(int argc, char **argv) {
                     for (int y = 0, k = 0; y < height; ++y) {
                         for (int x = 0; x < width; ++x, ++k) {
                             if (host_results[k]) {
-                                out.print("{:4d}, {:4d}\n", x, y);
+                                out.print("{:d}, {:d}\n", x, y);
```

Comment:
undo this

Reply:
Reverted `a850047453e8bb49ffd50cf9aab947368414f167`